### PR TITLE
Hud

### PR DESCRIPTION
**Pull request by patriciogonzalezvivo**
A HUD for raspberryPi

### CODE REVIEW

**Comment by blair1618** [comment]
I'm gonna have to take your word that this compiles and runs on the Pi, since we don't have build automation for it yet :)

This is probably not too big of a change to add, but in general we shouldn't be adding more to the Pi-specific files. What we want in the tangram-es repo is simply a minimal demonstration of using the core library on each platform. Handling specific hardware configurations is not the focus of this project, and other projects have already done that task far better than we could (e.g. GLFW, SDL) so we should use existing code for those aspects wherever we can. 

In particular, the next time we need to make changes on the Pi I think that we should strongly consider using SDL for the input and display surface management. It would make our own code smaller (more readable) and add functionality (like event-driven rendering). 

I'll make a few cosmetic changes and then merge this.

**Comment by patriciogonzalezvivo** [comment]
I'm compiling right now

**Comment by patriciogonzalezvivo** [comment]
I'm pushing one small correction and compiling your changes to check. I'm sorry for the inconvenience

**Comment by patriciogonzalezvivo** [comment]
Everything compiling upto here

File: core/include/fontstash-es [hunk 1]
```diff
@@ -1 +1 @@
-Subproject commit de034e4f4b5328f03b002ca4365353f6d08a83fe
+Subproject commit 5fecc158a64cd8bc7393f392ddc02191244e5e3d
```
Comment (karimnaaji) [inline core/include/fontstash-es]: Please do `git submodule update` every time you checkout to another branch

Comment (blair1618) [inline core/include/fontstash-es]: Good catch, I'll fix this real quick

**Pull request merged**